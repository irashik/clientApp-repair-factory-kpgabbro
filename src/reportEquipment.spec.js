import React from 'react';

import ReportEquipment  from './reportEquipment';

       


describe("<ReportEquipment/>", () => {
    test('should render correctly empty list', () => {

        const component = shallow(<ReportEquipment />)
        expect(component).toMatchSnapshot();
    });



    test.skip('should render correctly full list', () => {


        // добавь данные ...

        const component = shallow(<ReadModuleList />)

        expect(component).toMatchSnapshot();


    });




});



