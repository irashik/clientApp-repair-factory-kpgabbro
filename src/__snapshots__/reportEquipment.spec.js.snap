// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ReportEquipment/> should render correctly empty list 1`] = `
<Container
  fluid={true}
>
  <Row
    className="justify-content-md-center"
  >
    <h2>
      Отчет по оборудованию
    </h2>
  </Row>
  <Row
    className="justify-content-start"
  >
    <Col>
      <FormControl
        id="reportInputEquipment"
        idunit=""
        onChange={[Function]}
        placeholder="Выберите оборудование"
        size="sm"
        type="text"
        value=""
      />
      <SearchList
        filter=""
        onSelectEquipment={[Function]}
      />
    </Col>
    <Col>
      <Button
        active={false}
        disabled={false}
        onClick={[Function]}
        size="sm"
        variant="secondary"
      >
        Показать все записи
      </Button>
    </Col>
  </Row>
  <Row>
    <ShowReadModuleList
      idEquipment=""
      viewAllPosition={false}
    />
  </Row>
</Container>
`;
